feat(theme): add 'gruvbox-light' theme

mertssmnoglu opened the pull request:
## Description of change

I heard that flawz need some more themes #6. I tried something to implement a Vim Light Theme called [gruvbox-light](https://github.com/vim-scripts/gruvbox#light-mode) It's **NOT EXACT SAME** but it's looks like gruvbox-light.

## How has this been tested?
It can only be tested with the eyes. Testing in daylight is recommended.

Color Palette of `gruvbox-light`

![gruvbox-light color palette](https://camo.githubusercontent.com/fb51e4d6818d41160c5b181693947541c36c22cb2172e8f323386f5bfd07b089/687474703a2f2f692e696d6775722e636f6d2f3439714b7959572e706e67)

---

If we can agree on theme colors, I will add them too.

orhun reviewed (commented):
Very nice! I like it.

Can you also update README.md with a screenshot too?

mertssmnoglu commented:
I changed the colors of the selected line so that it can be highlighted more easily.

Also updated `README.md` with theme preview. Other previews are 1000x600 but I couldn't take the screenshot at the same resolution.

orhun reviewed (approved):
I updated the screenshot with the same resolution :)

Thanks for the contribution!
